src/components/flat-table/flat-table.spec.tsx

```diff
@@ -724,9 +726,45 @@ describe("FlatTable", () => {
             </FlatTableBody>
           </FlatTable>
         );
-        const tableWrapper = await screen.findByRole("region");
+        const tableWrapper = container.querySelectorAll(
+          "div"
+        )[1] as HTMLElement; // screen.findByRole("region");
         tableWrapper?.focus();
         expect(tableWrapper).toHaveFocus();
+        expect(await screen.findByRole("region")).toHaveStyle({
+          outline: "transparent 3px solid",
+          "box-shadow":
+            "0px 0px 0px var(--borderWidth300) var(--colorsSemanticFocus500),0px 0px 0px var(--borderWidth600) var(--colorsUtilityYin090)",
+        });
+        fireEvent.keyDown(tableWrapper, arrowDown);
+        expect(tableWrapper).toHaveFocus();
+      });
+
+      it("should not move focus to first row when down arrow pressed and table wrapper focused when focusRedesignOptOut is set", async () => {
+        const { container } = rtlRender(
+          <CarbonProvider focusRedesignOptOut>
+            <FlatTable>
+              <FlatTableBody>
+                <FlatTableRow onClick={() => {}}>
+                  <FlatTableCell>one</FlatTableCell>
+                  <FlatTableCell>two</FlatTableCell>
+                </FlatTableRow>
+                <FlatTableRow onClick={() => {}}>
+                  <FlatTableCell>three</FlatTableCell>
+                  <FlatTableCell>four</FlatTableCell>
+                </FlatTableRow>
+              </FlatTableBody>
+            </FlatTable>
+          </CarbonProvider>
+        );
+        const tableWrapper = container.querySelectorAll(
+          "div"
+        )[2] as HTMLElement; // screen.findByRole("region");
```

Comment:
question: same as before

But because there is two I'm probably just being silly and it's good to have?

Reply:
good spot I'll get these removed